docker/install.sh

```diff
@@ -106,7 +106,7 @@ cp bundle-22.1-SNAPSHOT.jar /opt/carma-simulation
 
 # Deploy scenario files
 cd /home/carma/src/co-simulation
-unzip sample_scenario.zip -d /opt/carma-simulation/scenarios
+unzip bundle/target/bundle-*.jar 'scenarios/*' -d /opt/carma-simulation/scenarios
```

Comment:
Looks like copies both the unzipped scenarios folder and its files to the location given. To replace it we need to update the destination path to /opt/carma-simulation/
With this we'll also need to supply an overwrite argument to unzip

Reply:
Thinking about this a little more, can we not use maven to copy the resources to that directory while building? 
There is this output resources tag which can be used to do that https://maven.apache.org/plugins/maven-resources-plugin/copy-resources-mojo.html#outputdirectory